### PR TITLE
Adding Peer Reviews to Sidebar

### PR DESCRIPTION
**Pull request by yattias**
Adding Peer Review section to bring more awareness to peer reviews

image

### CODE REVIEW

File: components/Home/sidebar/sidebar_items/PeerReviewSidebarItem.tsx
Comment (yattias) [inline components/Home/sidebar/sidebar_items/PeerReviewSidebarItem.tsx:1]: Following the template of bounty sidebar item. The structure below is mainly for consistency

File: next.config.js [hunk 1]
```diff
@@ -131,7 +131,9 @@ const SentryWebpackPluginOptions = {
   disableClientWebpackPlugin: false,
 };
 
-module.exports = withSentryConfig(
-  withTM({ ...nextConfig }),
-  SentryWebpackPluginOptions
-);
+module.exports = withTM({ ...nextConfig });
```
Comment (yattias) [inline next.config.js:134]: Temporarily commenting this out. We moved projects from Q5 account and they seem to not have been moved successfully for some reason. Commenting this out until its figured out

File: components/Home/sidebar/sidebar_items/PeerReviewSidebarItem.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+              </div>
+            )}
+            <span className={css(styles.reviewSidebarTitle)}>
+              <span>{"peer reviewed "}</span>
+            </span>
+          </div>
+          <div>
+            <StarInput value={score} readOnly size="small" />
+          </div>
+          <div className={css(styles.reviewSidebarItemContent)}>{docTitle}</div>
```
Comment (manveerxyz) [inline components/Home/sidebar/sidebar_items/PeerReviewSidebarItem.tsx:57]: nit: The bounties sidebar explicitly truncates titles longer than 180 characters ([source](https://github.com/ResearchHub/researchhub-web/blob/master/components/Home/sidebar/sidebar_items/BountiesSidebarItem.tsx#L83-L85)), we should probably do the same here.

**Review by manveerxyz — COMMENTED** [review]
Looks good, just one small thing!

**Pull request merged**